src/App.js

```diff
@@ -24,6 +32,8 @@ const App = () => (
       <span>Users: </span>
       {users.length}
     </p>
+
+    <PostList preparedPosts={preparedPosts} key={preparedPosts.id} />
```

Comment:
remove key, it's unnecessary here
we add keys only to the iterated elements inside loops